DiamondLightSource/python-artemis
# Make sure fake zocalo, used for developer testing, is tidied up well

Fixes #547

### To test:
1. Start then kill fake zocalo
2. Confirm `/home/$USER/.zocalo/` is cleaned
3. Start fake zocalo again (note will need a few seconds before stopping it) and confirm a new rabbit mq instance is created
